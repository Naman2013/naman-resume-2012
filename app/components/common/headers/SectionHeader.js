import React from 'react';
import PropTypes from 'prop-types';
import { lightTurqoise, black } from '../../../styles/variables/colors';

const SectionHeader = ({ title }) => (
  <div className="root">
    <h3 className="title">{title}</h3>

    <style jsx>{
      `
        .root {
          background-color: ${black};
          text-align: center;
        }

        .title {
          color: ${lightTurqoise}
        }
      `
    }</style>
  </div>
);

SectionHeader.defaultProps = {
  title: '',
};

SectionHeader.propTypes = {
  title: PropTypes.string,
};

export default SectionHeader;
