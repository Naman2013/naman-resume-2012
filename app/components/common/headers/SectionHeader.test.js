import React from 'react';
import renderer from 'react-test-renderer';
import SectionHeader from './SectionHeader';

const testData = {
  providedTitle: 'test title',
};

it('renders some text when provided', () => {
  const sectionHeaderComponent = renderer.create(
    <SectionHeader title={testData.providedTitle} />,
  );

  const renderedHeader = sectionHeaderComponent.toJSON();
  expect(renderedHeader).toMatchSnapshot();
});
