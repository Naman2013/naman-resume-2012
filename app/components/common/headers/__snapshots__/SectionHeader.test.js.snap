// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders some text when provided 1`] = `
<div
  className="jsx-96659203 root"
>
  <h3
    className="jsx-96659203 title"
  >
    test title
  </h3>
  
  <style
    jsx={true}
  >
    
        .root {
          font-family: "Brandon Grotesque", "brandon-grotesque", sans-serif;
          background-color: #000;
          text-align: center;
          padding: 10px 0;
        }

        .title {
          color: #b1cfcf;
          text-transform: none;
          font-size: 24px;
        }
    
  </style>
</div>
`;
